src/app/clusters/scenes-server/scenes-server.cpp

```diff
@@ -1061,13 +1070,13 @@ void ScenesServer::HandleCopyScene(HandlerContext & ctx, const Commands::CopySce
 
             ReturnOnFailure(AddResponseOnError(
                 ctx, response, sceneTable->SetSceneTableEntry(ctx.mCommandHandler.GetAccessingFabricIndex(), scene)));
-        }
 
-        // Update SceneInfoStruct Attributes
-        ReturnOnFailure(
-            AddResponseOnError(ctx, response,
-                               UpdateFabricSceneInfo(ctx.mRequestPath.mEndpointId, ctx.mCommandHandler.GetAccessingFabricIndex(),
-                                                     Optional<GroupId>(), Optional<SceneId>(), Optional<bool>())));
+            // Update SceneInfoStruct Attributes after each insert in case we hit max capacity in the middle of the loop
+            ReturnOnFailure(AddResponseOnError(ctx, response,
+                                               UpdateFabricSceneInfo(ctx.mRequestPath.mEndpointId,
+                                                                     ctx.mCommandHandler.GetAccessingFabricIndex(),
+                                                                     Optional<GroupId>(), Optional<SceneId>(), Optional<bool>())));
```

Comment:
please add a `/* = name*` for the bool as bools are not obvious what they represent. I can see that the group and scene are missing ... what is the bool?